src/components/UserForm.js

```diff
@@ -125,20 +125,24 @@ const UserForm = ({ variant, standalone = true, onSubmitFulfilled }) => {
           onChange={handleChange("username")}
           InputLabelProps={{ shrink: true }}
         />
-        <TextField
-          variant="outlined"
-          margin="normal"
-          required
-          fullWidth
-          id="email"
-          label="Email"
-          name="email"
-          placeholder="unicorn@usepow.app"
-          value={state.email}
-          autoComplete="email"
-          onChange={handleChange("email")}
-          InputLabelProps={{ shrink: true }}
-        />
+        {variant === "signup" ? (
+          <TextField
+            variant="outlined"
+            margin="normal"
+            required
+            fullWidth
+            id="email"
+            label="Email"
+            name="email"
+            placeholder="unicorn@usepow.app"
+            value={state.email}
+            autoComplete="email"
+            onChange={handleChange("email")}
+            InputLabelProps={{ shrink: true }}
+          />
+        ) : (
+          "current-email"
```

Comment:
Nothing should happen if login. Try using `variant === "signup" && <Text...`

Reply:
I did what you said and 
POW!👊
Works!






![POW-POW!👊_Works!-126](https://user-images.githubusercontent.com/7642493/76492184-e36c7d80-642f-11ea-83d8-2a6363e74ec3.jpg)